ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/frontend.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <frontend_manager/frontend_manager.hpp>
+#include "exceptions.hpp"
+#include "model.hpp"
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        class PDPD_API FrontEndPDPD : public FrontEnd
+        {
+            static std::shared_ptr<Function>
```

Comment:
Why do we need to have this private static method?

Reply:
Convertion of model for PDPD frontend looks not dependent on any member variable, thus it was made static.
I guess it was just a "developer's preference", it could be 'non-static and const', but Frontend implementation for PDPD case doesn't have any class members